src/components/cart.js

```diff
@@ -27,6 +27,7 @@ export default class Cart extends Component {
     this.childTemplate = new Template(this.config.lineItem.templates, this.config.lineItem.contents, this.config.lineItem.order);
     this.node = config.node || document.body.appendChild(document.createElement('div'));
     this.isVisible = this.options.startOpen;
+    this.cache = null;
```

Comment:
more descriptive name. Without reading more coed, I don't know what's being cached here. If it's line items, maybe `lineItemCache`?